src/styles/_catalog.scss

```diff
@@ -1,22 +1,33 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: $cards-gap;
+  grid-template-columns: repeat(auto-fill, $card-width);
+  justify-content: center;
+  max-width: $four-cards-container;
 
-  max-width: 944px;
-  margin: 0 auto;
+  &--margins--bottom {
+    @include margins(auto);
+
+    @media (max-width: 1023.9px) {
+      grid-template-columns: repeat(3, $card-width);
+    }
+
+    @media (max-width: 767.9px) {
+      grid-template-columns: repeat(2, $card-width);
+    }
+
+    @media (max-width: 487.9px) {
```

Comment:
Where are these numbers from?

Reply:
![image](https://user-images.githubusercontent.com/105124305/188803696-bb73cc6f-3da0-43d5-98e8-afae0afbb154.png)
because of task. If I put 488px it will not change on two columns! 

![image](https://user-images.githubusercontent.com/105124305/188803878-fe851fab-ed40-4a45-9be9-c15df6f39dbf.png)
same here!

![image](https://user-images.githubusercontent.com/105124305/188804020-64f9739d-78da-4fc1-abaa-7789f442e170.png)
same here!

Reply:
I saw one guy in Youtube he use such strange technique and it works. It's strange because I ever see screen 487.5px or smth like that, but in general for me it's more understandable that my media query is about 1024 when I use 1023.9 razer then I used to use 1023.